pkg/projects/descriptions.json

```diff
@@ -36,7 +36,7 @@
     },
     {
         "Name": "Microsoft.NETCore.UniversalWindowsPlatform",
-        "Description": "Provides a set of packages that can be used when building Universal Windows applications on .NETCore.",
+        "Description": "Provides a set of packages that can be used when building Universal Windows applications on .NETCore.\nWhen using Visual Studio this package requires Visual Studio 2017 or later.",
```

Comment:
For context: Our Microsoft.Net.Native.Compilers package is only supported on Visual Studio 2017 or later. Since the UWP metapackage is now responsible for bringing this package in, we want to make sure developers using Visual Studio 2015 are aware that upgrading to 5.3.x will not work for them.